cmd2/utils.py

```diff
@@ -317,6 +344,11 @@ def clear(self) -> None:
         """Clear the internal contents"""
         self.buffer.byte_buf = b''
```

Comment:
If you are changing internal representation to a bytearray, then this needs to be changed too